wsd/ClientRequestDispatcher.cpp

```diff
@@ -9,10 +9,15 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+#include <Poco/Net/HTTPRequest.h>
+#include <Poco/Net/HTTPResponse.h>
```

Comment:
We don't want to use Poco ever if we can avoid it for various reasons.
see net/HttpRequest.hpp - and 'asyncRequest' =)